package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chauffeur-prive/node-amqp-bus",
-  "version": "5.0.0",
+  "version": "5.0.1",
```

Comment:
If you do this change, it's minor ~major fix.

Reply:
I agree with @TimotheeMalard

Reply:
Why ?

Reply:
I agree with @jean-k-cp ⚡️